tests/unit/test_launchers.py

```diff
@@ -63,11 +76,15 @@ def test_stop_fail(self):
 
 
 class TestMozRunnerLauncher(unittest.TestCase):
+    def clean_launcher(self):
+        if hasattr(self, 'launcher'):
+            del self.launcher
```

Comment:
I generally don't really like relying on destructors in python for behavior, as it's kind of unintuitive. If there's cleanup code that needs to be run when we're finished, I'd rather have a cleanup() method that's called explicitly.